index.html

```diff
@@ -5,6 +5,75 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+
```

Comment:
Please apply nice formatting to the document. It's expected to have beautiful indents and only necessary blank lines